src/main/java/brooklyn/entity/proxy/aws/ElbControllerImpl.java

```diff
@@ -555,20 +597,23 @@ protected String generateUnusedElbName() {
             elbName = new JcloudsMachineNamer().generateNewGroupId(setup);
             boolean exists = doesLoadBalancerExist(elbName);
             if (!exists) return elbName;
-            LOG.debug("Auto-generated ELB name {} in {} conflicts with existing; trying again (attempt {}) to generate name", new Object[] {elbName, this, (i+2)});
+            LOG.debug("Auto-generated ELB name {} in {} conflicts with existing; " +
+                    "trying again (attempt {}) to generate name", new Object[]{elbName, this, (i+2)});
         }
-        throw new IllegalStateException("Failed to unused auto-genreate ELB name after "+maxAttempts+" attempts (last attempt was "+elbName+")");
+        throw new IllegalStateException(
+            "Failed to unused auto-genreate ELB name after "+maxAttempts+" attempts (last attempt was "+elbName+")");
```

Comment:
I know you didn't write it but this message doesn't make any sense. I guess it should be "Failed to auto-generate  unused ELB name ..."